ruby-challenges.rb

```diff
@@ -33,4 +56,19 @@
 palindrome_test_case3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
 
-# Pseudo code:
\ No newline at end of file
+# Pseudo code:
+# create a method called palindrome_check using string as a parameter
+    # create a conditional using string.downcase to turn the string into all lowercase then == string.downcase.reverse to do the same but adding the reverse method.
+    #use string interpolation to call the string. 
+    # LEARN becomes nrael
+    #print the results calling the method and the variable as an argument
+def palindrome_check (string)
+    if string.downcase == string.downcase.reverse
+        "#{string} is a palindrome"
+    else 
+        "#{string} is not a palindrome"
+    end
+end
+p palindrome_check palindrome_test_case1
+p palindrome_check palindrome_test_case2
+p palindrome_check palindrome_test_case3
```

Comment:
You crushed the Ruby section!  Don't forget your built-in methods on the first problem.